rts/lra-examples/lra-jwt/wildfly/client/src/main/java/org/jboss/jaxrsjwt/client/JwtRestClient.java

```diff
@@ -0,0 +1,71 @@
+package org.jboss.jaxrsjwt.client;
+
+import org.jboss.jaxrsjwt.auth.JwtManager;
+
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.Form;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+public class JwtRestClient {
+    private static final String AUTHZ_HEADER = "Authorization";
+    private static final String REST_TARGET_URL = "http://localhost:8080/lra-coordinator/lra-coordinator";
+    private static final String ADMIN_ROLE = "admin";
+    private static final String PARTICIPANT_ROLE = "participant";
+
+    private String role;
+
+    public JwtRestClient() {
+        role = null;
+    }
+
+    public JwtRestClient(String role) {
+        this.role = role;
+    }
+
+    public static void main(String[] args) throws Exception{
+        //participant role
+        JwtRestClient participantClient = new JwtRestClient(PARTICIPANT_ROLE);
+        participantClient.test(true);
+        //admin role
+        JwtRestClient adminClient = new JwtRestClient(ADMIN_ROLE);
+        adminClient.test(true);
+        //test without token
+        JwtRestClient noToken = new JwtRestClient();
+        noToken.test(false);
+
+    }
+
+    public void test(boolean obtainToken) throws Exception{
+        System.out.println("------------------------------");
+        System.out.printf("Testing %s ",(role != null ? role : "without token"));
+        System.out.println("------------------------------");
+        String authzHeaderValue = null;
+        if (obtainToken) {
+            System.out.println("Obtaining JWT...");
+            JwtManager jwtManager = new JwtManager();
+            String jwt = jwtManager.createJwt(role, role);
+            if (jwt == null) {
```

Comment:
I don't think this can happen. When the `jwt` could be `null`?

Reply:
If the keystore is missing, then this null check could help

Reply:
Please, elaborate on this point. I can't find any connection to the missing keystore with the returned null.
Ideally the part of the code where the object could be set as `null`.

Reply:
If user forgets to put the `jwt.keystore` file in the `resources` folder then [signer](https://github.com/jbosstm/quickstart/pull/300/files#diff-df63fbed0a82c3671be6ee102b7cae6ec8c0b4247cc7162dcef0a0cebe45946dR76) will be null and there will be no JWT token generated.

Reply:
I'm sorry but I'm not getting this. I'm looking at the code and I can't find a way how the `createJwt` method may return `null` (https://github.com/jbosstm/quickstart/pull/300/files#diff-df63fbed0a82c3671be6ee102b7cae6ec8c0b4247cc7162dcef0a0cebe45946dR92)

Reply:
Okay, sorry Ondra, I was thinking something else. You are correct it will not return `null`. 
I will remove the null check. Thanks